hx_lti_initializer/templates/ig/detail.html

```diff
@@ -53,9 +53,11 @@
 				context_id: "{{ course }}",
 				collection_id: "{{ collection }}",
 				object_id: {{ object }},
+				//Uhhh....this token is exposed -- TODO!!
 				token: "{{ token }}",
 				username: "{{ username }}",
 				user_id: "{{ user_id }}",
+				instructor_id: '{{ instructor_id }}'
```

Comment:
Note: this is making an assumption that there is 1 instructor for the course. Is that a safe assumption?